deebot_client/commands/json/network.py

```diff
@@ -35,3 +38,27 @@ def _handle_body_data_dict(
             )
         )
         return HandlingResult.success()
+
+
+class GetNetInfoLegacy(JsonCommandWithRawMessageHandling):
```

Comment:
```suggestion
class GetNetInfoLegacy(JsonCommand, CommandWithMessageHandling, MessageDict):
```

As the legacy command is an exception and no other command uses the "raw" format, we don't need a class for it. We can always add it later if needed